packages/gatsby-source-filesystem/src/gatsby-node.js

```diff
@@ -171,6 +181,7 @@ See docs here - https://www.gatsbyjs.org/packages/gatsby-source-filesystem/
   })
 
   watcher.on(`change`, path => {
+    console.log(path)
```

Comment:
Intentional?

Reply:
This does look like debugging helper, which should be removed